iam.tf

```diff
@@ -0,0 +1,44 @@
+resource "aws_iam_role" "ranger_task_exec" {
+  name = "ranger-ecs-task-exec-${var.aws_region}"
+
+  assume_role_policy = <<EOF
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Sid": "",
+      "Effect": "Allow",
+      "Principal": {
+        "Service": "ecs-tasks.amazonaws.com"
+      },
+      "Action": "sts:AssumeRole"
+    }
+  ]
+}
+EOF
+}
+
+resource "aws_iam_role_policy_attachment" "task_exec_managed" {
```

Comment:
rename resource to `ranger_task_exec_policy`. Makes more readable

Reply:
done